test/e2e/testing_manifests/default_irq_smp_affinity.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: openshift-default-irq-smp-affinity
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=An OpenShift profile to test [scheduler] default_irq_smp_affinity option
+      include=openshift-node
+      [scheduler]
+      # isolated_cores take a list of ranges; e.g. isolated_cores=2,4-7
+      isolated_cores=1
+      # the cpulist in 'default_irq_smp_affinity' is unpacked and written directly to /proc/irq/default_smp_affinity
+      default_irq_smp_affinity=1
```

Comment:
OK, that sets the default_irq_smp_affinity differently from the default functionality of the tuned.
I would add a test ensuring the prev functionality still works the same (not setting default_irq_smp_affinity results in the inverse mask of the isolated_cores)

Reply:
I'll see what I can do.

Reply:
is only a thought, nothing more

Reply:
Done.